src/java/org/apache/cassandra/db/compression/CompressionDictionaryManager.java

```diff
@@ -267,6 +336,11 @@ private void handleNewDictionary(CompressionDictionary dictionary)
         onNewDictionaryTrained(dictionary.dictId());
     }
 
+    public void importDictionary(CompressionDictionary dictionary)
```

Comment:
~this method could be `private`~

Actually, should you just remove the method? I think the only call-site `CompressionDictionaryManager#importCompressionDictionary` can call `handleNewDictionary` instead.